geps/gep-1897/index.md

```diff
@@ -49,6 +46,20 @@ this proposal.
 
 ## Non-Goals
 
+A major non-goal for this proposal is for configuration of the service mesh transport security.
+It is very common for service mesh implementations to implement some form of transparent transport security, whether that is WireGuard, mTLS, or others.
+This is completely orthogonal to the use cases being tackled by this GEP.
+
+* The "mesh transport security" is something invisible to the users application, and is simply used to secure communication between components in the mesh.
+* This proposal, instead, explicitly calls for sending TLS **to the users application**.
```

Comment:
Sorry. What's the "user's application" in this context? I imagine it's an application that an Application Developer (Ana) develops and deploys to the mesh. Just to be clear who the "user" is here.

```suggestion
* The "mesh transport security" is something invisible to the user's application, and is simply used to secure communication between components in the mesh.
* This proposal, instead, explicitly calls for sending TLS **to the user's application**.
```

Reply:
AFAIU user application means the application process, and not the sidecar/node-proxy

Reply:
Please use Gateway API terminology to explain this.